### DISCUSSION TITLE
Rig control and Icom (Resolved)

### DISCUSSION THREAD

**Discussion by f4emm** [foldynl/QLog#114] [General]
Hello,
I try to connect my IC-7610 and my IC-9700 without success.
Yet everything is ok with wsjt-x, ROS...
I'm thinking of the CI-V.
Does anyone have the solution?
Thanks
Have a good day

**Comment by foldynl** [comment]
if I can help you, they need to give me the details of what happened.

- your OS. If it is linux then the exact version.
- I need an error what you got(screenshot or plain text)
- your description what "try to connect without success" means.You get an error or what happened
- your setting in Setting dialog.The best way is to send a screenshot.

I have ic7300 and it works without any problem.Then it is tested against K2 and K3. CI-V should work without any problem.

**Reply by f4emm** [reply]
Hello,
Attached are two photos.
Windows 8.1 system.
Thanks
Have a good day

https://zupimages.net/viewer.php?id=22/37/jmzy.png

https://zupimages.net/viewer.php?id=22/37/utly.png

**Comment by foldynl** [comment]
thanks for sharing. Taking a quick look, I see only one mistake so far. If you have a port, please use COM2. I see that it is necessary to display it a little differently for Window. Thanks, I'll try to improve it.

**Reply by f4emm** [reply]
It works !
Thanks a lot
Have a good day
F4EMM 73's

**Comment by foldynl** [comment]
Many thanks for feedback. Your feedback improves next versions
